src/Sentry.AspNetCore/ScopeExtensions.cs

```diff
@@ -87,7 +87,9 @@ public static void Populate(this Scope scope, HttpContext context, SentryAspNetC
                         scope.SetTag("route.area", area);
                     }
 
-                    scope.Transaction = area == null ? $"{controller}.{action}" : $"{area}.{controller}.{action}";
+                    scope.TransactionName = area == null
```

Comment:
Can we have this in some internal static method to make sure changing its stays aligned with the one on the middleware?